api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -231,6 +231,30 @@ message AntennaState {
   antenna.AntennaPolarization polarization = 3;
 }
 
+// The current state of the plan being executed by the ground station.
+message PlanState {
+  // The status of the plan.
+  enum Status {
+    UNKNOWN = 0;
+    STARTING = 1;
```

Comment:
What is `STARTING`? Should there be `COMPLETING` too?

Reply:
Ah, 'STARTED' seems better.